config/routes.rb

```diff
@@ -24,6 +25,7 @@
   post '/Patient/:patient_id/$health-cards-issue', to: 'health_cards#create', as: :issue_vc, format: :fhir_json
   get '/Patient/:id', to: 'patients#show', as: :fhir_patient, format: :fhir_json
   get '/Immunization/:id', to: 'immunizations#show', as: :fhir_immunization, format: :fhir_json
+  get "/LabResult/:id", to: "lab_results#show", as: :fhir_lab_result, format: :fhir_json
```

Comment:
`LabResult` is not a FHIR resource type.

Reply:
resolved